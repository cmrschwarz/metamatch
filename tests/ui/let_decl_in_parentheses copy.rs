use metamatch::unquote;
fn main() {
    unquote! {
        (let foo = 3) + 3
    }
}
