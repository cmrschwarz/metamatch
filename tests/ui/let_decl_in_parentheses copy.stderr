error: `let` cannot occur within expression
 --> tests/ui/let_decl_in_parentheses copy.rs:4:10
  |
4 |         (let foo = 3) + 3
  |          ^^^
